hw-7/task1.js

```diff
@@ -0,0 +1,13 @@
+
+function handleNum (number, fun1, fun2) {
```

Comment:
use informative name for parameters,for example `handleEven` `handleOdd` or `onEven` `onOdd`